src/app/@core/helpers/fuentes/fuenteHelper.ts

```diff
@@ -35,10 +35,13 @@ export class FuenteHelper {
                         this.pUpManager.showErrorAlert('No se pudo consultar las fuentes');
                         return undefined;
                     } else if (!id || this.query_params === '0/1') {
-                        res.forEach(element => {
-                            element.Vigencia === 0 ? element.Vigencia = 'sin vigencia asignada' : element.Vigencia;
-                        });
+                        if (res && res.length > 0) {
+                            res.forEach(element => {
+                                element.Vigencia === 0 ? element.Vigencia = 'sin vigencia asignada' : element.Vigencia;
```

Comment:
Revisar la L40:
- No se está usando el valor retornado por la terna
- Es posible negar la lógica para invertir las ramas de la terna, y para aprovechar 
  ![image](https://user-images.githubusercontent.com/6588872/163752453-75b363f9-aa8f-4768-8276-60bf367db3ff.png)
- Adicionalmente, cuando `element.Vigencia !== 0`, por lo primero, no estaría pasando nada
- Falta posiblemente algun par de paréntesis para desambiguar los miembros de la terna

Revisar si se puede reescribir así:
```suggestion
                                element.Vigencia = element.Vigencia ? element.Vigencia : 'sin vigencia asignada';
```

Reply:
Se realiza el cambio debido a que el funcionamiento aparentemente es el mismo